website/client/readme.md

```diff
@@ -1,20 +0,0 @@
-#Running

```

Comment:
I'm trying to add the file again but it won't let me because there is already a file with the title README.md that has the same contents. Is it necessary to have both?